File: csharp/dsf/dsf/src/register/MsgSvcState.cs
namespace com.baidu.dsf.register
{

    /// <summary>
    /// 服务状态消息
    /// {
    ///  "name" : "online",      //状态名
    ///  "value":"true"          //状态值
    /// }
    /// @author arksea
    /// </summary>
    public class MsgSvcState
    {

        public readonly string name;
        public readonly string value;

        public MsgSvcState(string name, string value)
        {
            this.name = name;
            this.value = value;
        }
    }

}

File: java/dsframework/src/main/java/dsf/register/MsgSvcState.java
package dsf.register;

/**
 * 服务状态消息
 * {
 *  "name" : "online",      //状态名
 *  "value":"true"          //状态值
 * }
 * @author arksea
 */
public class MsgSvcState {

    public final String name;
    public final String value;

    public MsgSvcState(String name, String value) {
        this.name = name;
        this.value = value;
    }
}
